cypress/e2e/inventory/holdings/verify-holdings- source-added-holdings-manually.cy.js

```diff
@@ -0,0 +1,66 @@
+import { including } from '@interactors/html';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import Users from '../../../support/fragments/users/users';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryNewHoldings from '../../../support/fragments/inventory/inventoryNewHoldings';
+
+const testData = {
+  user: {},
+};
+const item = {
+  instanceName: `Inventory Instance ${randomFourDigitNumber()}`,
+  itemBarcode: randomFourDigitNumber(),
+};
+const permanentLocation = 'Annex (KU/CC/DI/A)';
+const calloutMessage = 'has been successfully saved.';
+
+describe('Holdings', () => {
+  before('Create test data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiInventoryViewCreateEditHoldings,
+      Permissions.uiInventoryViewCreateEditItems,
+    ]).then((createdUserProperties) => {
+      testData.user = createdUserProperties;
+      item.instanceId = InventoryInstances.createInstanceViaApi(
+        item.instanceName,
+        item.itemBarcode,
+      );
+      cy.login(testData.user.username, testData.user.password, {
+        path: TopMenu.inventoryPath,
+        waiter: InventoryInstances.waitContentLoading,
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    Users.deleteViaApi(testData.user.userId);
+  });
+
+  it(
+    'C367932 Verify holdings source added holdings manually (firebird) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.firebird] },
+    () => {
+      InventoryInstance.searchByTitle(item.instanceName);
+      InventorySearchAndFilter.verifyInstanceDisplayed(item.instanceName);
+      InventorySearchAndFilter.checkRowsCount(1);
+
+      InventoryInstances.selectInstance();
+      InventoryInstance.waitLoading();
+
+      InventoryInstance.pressAddHoldingsButton();
+      InventoryNewHoldings.fillRequiredFields(including(permanentLocation));
+      InventoryNewHoldings.checkPermanentLocation(permanentLocation);
```

Comment:
use `changePermanentLocation` from `cypress\support\fragments\inventory\holdingsRecordEdit.js` instead of line 57 and 58